README.md

```diff
@@ -1 +1,36 @@
-# Webpack_demo
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo List app
+
+> This app allow you to track and manage your 'To Do' tasks in an interactive and simple-to-use way.
+
+
+## 🧰 Built With
+
+- HTML
+- CSS
+- JavaScript
+-Webpack
+
+## Authors
+
+👤 **Camilo Vela**
+
+- GitHub: [@camilovelag](https://github.com/camilovelag)
+- Twitter: [@camilovelag](https://twitter.com/camilovelag)
+- LinkedIn: [camilovelag](https://linkedin.com/in/camilovelag)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add an MIT.md file to your root folder so that this link is not broken. You can get an [MIT template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) from the Microverse template [here](https://github.com/microverseinc/readme-template/blob/master/MIT.md).

Reply:
Done!